src/Helper/Quote.php

```diff
@@ -158,11 +157,15 @@ protected function setItemQty()
      */
     protected function setPayment()
```

Comment:
Add to description:
```
If it's an online method, the default Shopgate payment method is used as it will not trigger capture when the quote turns into an order. We manipulate the online payment method after the order is created.
```